dense_vector/track.py

```diff
@@ -72,69 +74,73 @@ def __init__(self, track, params, **kwargs):
         with open(os.path.join(cwd, "queries.json"), "r") as file:
             lines = file.readlines()
         self._queries = [json.loads(line) for line in lines]
-        self._iters = 0
         self.infinite = True
 
     def partition(self, partition_index, total_partitions):
         return self
 
     def params(self):
-        result = {"index": self._index_name, "cache": self._params.get("cache", False), "size": self._params.get("k", 10)}
-        result["script_body"] = {
-            "query": {
-                "script_score": {
-                    "query": {"match_all": {}},
-                    "script": {
-                        "source": "cosineSimilarity(params.query, 'vector') + 1.0",
-                        "params": {"query": self._queries[self._iters]},
-                    },
-                }
-            },
-            "_source": False,
-        }
-        result["knn_body"] = {
-            "knn": {
-                "field": "vector",
-                "query_vector": self._queries[self._iters],
-                "k": self._params.get("k", 10),
-                "num_candidates": self._params.get("num-candidates", 100),
-            },
-            "_source": False,
+        return {
+            "index": self._index_name,
+            "cache": self._params.get("cache", False),
```

Comment:
Why are we defaulting to `cache` being false? The underlying dataset isn't changing, so caching the result set is perfectly fine IMO. Is there something I am missing?

Reply:
I pulled these settings from the latency/throughput runs for knn search where I assume caching could change our performance test results in an undesirable way. For this test, we don't use the same query vector twice (or general query), so wouldn't caching potentially use additional resources that end up being wasted? It's certainly possible my understanding isn't correct here, though.